Tests/SUITCaseTests/CompareImagesTests.swift

```diff
@@ -76,37 +76,29 @@ class CompareImagesTests: XCTestCase {
     }
 
     func testStrictComparison() {
+        let referenceImage = UIImage(named: "en_iPhone_X_reference_strict", in: .module, compatibleWith: nil)!
```

Comment:
Looks like we reuse these images in all tests. Can you please 

1. Reduce code repetition by moving these variables from the test cases to the test classes?
2. Double-check that assets only include the images we need.